Docs for primitive key support (#4478)
confluentinc/ksql: docs-md/tutorials/basics-local.md

Your output should resemble:

```
 Kafka Topic                 | Partitions | Partition Replicas
--------------------------------------------------------------
 default_ksql_processing_log | 1          | 1
 pageviews                   | 1          | 1
 users                       | 1          | 1
--------------------------------------------------------------
```

> By default, KSQL hides internal / system topics. Use the SHOW ALL TOPICS

Reviewer: ```suggestion
By default, KSQL hides internal and system topics. Use the SHOW ALL TOPICS
```

statement to see the full list of topics in the Kafka cluster:

```sql
SHOW ALL TOPICS;
```

Your output should resemble:

```
  Kafka Topic                            | Partitions | Partition Replicas 
--------------------------------------------------------------------------
 __confluent.support.metrics            | 1          | 1                  
 _confluent-ksql-default__command_topic | 1          | 1                  
 _confluent-license                     | 1          | 1                  
 _confluent-metrics                     | 12         | 1                  
 default_ksql_processing_log            | 1          | 1                  
 pageviews                              | 1          | 1                  
 users                                  | 1          | 1                  
--------------------------------------------------------------------------
```

Inspect the `users` topic by using the PRINT statement:

```sql
PRINT users;
```

!!! note

>    The PRINT statement is one of the few commands in ksqlDB that are

Reviewer: ```suggestion
   The PRINT statement is one of the few case-sensitive commands in ksqlDB,
```

>    case-sensitive, even when the topic name is not quoted.

Reviewer: ```suggestion
   even when the topic name is not quoted.
```

    !!! tip
          You can run `DESCRIBE pageviews_original;` to see the schema for the

>           stream. Notice that KSQL created an additional columns, named

Reviewer: ```suggestion
          stream. Notice that ksqlDB created an additional column, named
```

>           `ROWTIME`, which corresponds with the Kafka message timestamp.

Reviewer: ```suggestion
          `ROWTIME`, which corresponds with the {{ site.ak }} message timestamp.
```

    ---------------
    ```

    !!! note
          You may have noticed the CREATE TABLE did not define the set of columns
          like the CREATE STREAM statement did. This is because the value format

>           if Avro, and the DataGen tool publishes the Avro schema to the {{ site.sr }}.

Reviewer: ```suggestion
          is Avro, and the DataGen tool publishes the Avro schema to {{ site.sr }}.
```

>           ksqlDB retrieves the schema from the Schema Registry and uses this to build

Reviewer: ```suggestion
          ksqlDB retrieves the schema from {{ site.sr }} and uses this to build
```

          the SQL schema for the table. You may still provide the schema if you wish.
          Until [Github issue #4462](https://github.com/confluentinc/ksql/issues/4462)
          is complete, schema inference is only available where the key of the data
          is a STRING, as is the case here.

    !!! note

>           The data generated has the same value in the Kafka record's key

Reviewer: ```suggestion
          The data generated has the same value in the {{ site.ak }} record's key
```

>           as is in the userId field in the value. Specifying `key='userId'`

Reviewer: ```suggestion
          as the `userId` field in the value. Specifying `key='userId'`
```

>           in the WITH clause above lets ksqlDB know this. This information

Reviewer: ```suggestion
          in the WITH clause above lets ksqlDB know this. ksqlDB uses this information
```

>           will be used later allow joins against the table to use the more

Reviewer: ```suggestion
          to allow joins against the table to use the more
```

          descriptive `userId` column name, rather than `ROWKEY`. Joining

>           on either will yield the same results. If your data does not

Reviewer: ```suggestion
          on either yields the same results. If your data doesn't
```

>           contain a copy of the key in the value simply join on `ROWKEY`.

Reviewer: ```suggestion
          contain a copy of the key in the value, you can join on `ROWKEY`.
```

    ```
    ksql> SHOW STREAMS;

     Stream Name         | Kafka Topic                 | Format    
    ---------------------------------------------------------------
     KSQL_PROCESSING_LOG | default_ksql_processing_log | JSON      
     PAGEVIEWS_ORIGINAL  | pageviews                   | DELIMITED 
    ---------------------------------------------------------------

    ksql> SHOW TABLES;

     Table Name     | Kafka Topic | Format | Windowed 
    --------------------------------------------------
     USERS_ORIGINAL | users       | AVRO   | false    
    --------------------------------------------------
    ```

!!! tip
    Notice the `KSQL_PROCESSING_LOG` stream listed in the SHOW STREAMS

>     output? KsqlDB will append messages detailing any issues it

Reviewer: ```suggestion
    output? ksqlDB appends messages that describe any issues it
```

>     encountered while processing your data. If things are not working

Reviewer: ```suggestion
    encountered while processing your data. If things aren't working
```

>     as you expect it can be worth checking the contents of this stream

Reviewer: ```suggestion
    as you expect, check the contents of this stream
```
